deploy/packages/bbp-packages.yaml

```diff
@@ -44,6 +44,7 @@ packages:
       #- python
     specs:
       - parquet-converters@0.3
+      - placement-algorithm@2.0.0
```

Comment:
It's primary usage is a set of scripts; but it can be used as a library as well (that's why `app` variant in the recipe).
Does the syntax support aliasing (i.e. naming module differently than package?). If not, I'll follow your suggestion to rename the package.